src/tasks-handler.js

```diff
@@ -0,0 +1,39 @@
+/* eslint-disable no-plusplus */
+/* eslint-disable radix */
+
+class Crud {
+  // Method to add a new task to tasklist.
+  static addTask(list, input) {
+    if (input !== '') {
+      const task = {
+        description: input,
+        completed: false,
+        index: list.length,
+      };
+      list.push(task);
+    }
+  }
+
+  // Method to remove a task from tasklist.
+  static removeTask(list, i) {
+    if (i >= 0) {
+      const lastitem = list.length - 1;
+      if (i === lastitem) {
+        list.splice(i, 1);
+      } else {
+        list.splice(i, 1);
+        let j = parseInt(i);
```

Comment:
Here you can just place `let j = parseInt(i, 10)` and remove the `/*eslint-disable radix*/` from line 2.